### PR TITLE
Default to 60hz if the driver doesn't return a usable value

### LINKED ISSUE #644: Fails to start with runtime error: division by zero
Hi,

I've compiled an up-to-date master from source on a debian buster machine.
When i try to run the binary, it immediately crashes with this error:
```

/data/diablo/source/devilutionX/SourceX/display.cpp:107:25: runtime error: division by zero 
AddressSanitizer:DEADLYSIGNAL
=================================================================
==3816==ERROR: AddressSanitizer: FPE on unknown address 0x55b1beb047bc (pc 0X55b1beb047bc bp 0x7ffedf25e260 sp 0x7ffedf25e0f0 T0)
    #0 0x55b1beb047bb in dvl::SpawnWindow(char const*, int, int) /data/diablo/source/devilutionX/SourceX/display.cpp:107
    #1 0x55b1bedbc1e6 in init_create_window /data/diablo/source/devilutionX/Source/init.cpp:47
    #2 0x55b1bec8fc35 in dvl::diablo_init() /data/diablo/source/devilutionX/Source/diablo.cpp:255
    #3 0x55b1bec8fda0 in DiabloMain /data/diablo/source/devilutionX/Source/diablo.cpp:297
    #4 0x55b1bebbb4c0 in main /data/diablo/source/devilutionX/SourceX/main.cpp:28
    #5 0x7fde2896409a in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2409a)
    #6 0x55b1bead77c9 in _start (/data/diablo/devilutionx+0xd287c9)

AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: FPE /data/diablo/source/devilutionX/SourceX/display.cpp:107 in dvl::SpawnWindow(char const*, int, int)
==3816==ABORTING

```

The error occurs in both fullscreen and windowed ( -x parameter) mode.

The SDL version i'm using:

```
ii  libsdl-image1.2:amd64                                       1.2.12-10+deb10u1                                       amd64        Image loading library for Simple DirectMedia Layer 1.2, libraries
rc  libsdl-mixer1.2:amd64                                       1.2.12-3                                                amd64        Mixer library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-net1.2:amd64                                         1.2.8-6                                                 amd64        Network library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-sound1.2:amd64                                       1.0.3-9                                                 amd64        Sound library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-ttf2.0-0:amd64                                       2.0.11-6                                                amd64        TrueType Font library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl1.2debian:amd64                                       1.2.15+dfsg2-4                                          amd64        Simple DirectMedia Layer
ii  libsdl1.2debian:i386                                        1.2.15+dfsg2-4                                          i386         Simple DirectMedia Layer
ii  libsdl2-2.0-0:amd64                                         2.0.9+dfsg1-1                                           amd64        Simple DirectMedia Layer
ii  libsdl2-2.0-0:i386                                          2.0.9+dfsg1-1                                           i386         Simple DirectMedia Layer
ii  libsdl2-dev                                                 2.0.9+dfsg1-1                                           amd64        Simple DirectMedia Layer development files
ii  libsdl2-mixer-2.0-0:amd64                                   2.0.4+dfsg1-1                                           amd64        Mixer library for Simple DirectMedia Layer 2, libraries
ii  libsdl2-mixer-dev:amd64                                     2.0.4+dfsg1-1                                           amd64        Mixer library for Simple DirectMedia Layer 2, development files
ii  libsdl2-ttf-2.0-0:amd64                                     2.0.15+dfsg1-1                                          amd64        TrueType Font library for Simple DirectMedia Layer 2, libraries
ii  libsdl2-ttf-dev:amd64                                       2.0.15+dfsg1-1                                          amd64        TrueType Font library for Simple DirectMedia Layer 2, development files

```

### PR DESCRIPTION
**Pull request by AJenbo**
This fixes #644

### CODE REVIEW

File: SourceX/display.cpp [hunk 1]
```diff
@@ -104,7 +104,7 @@ bool SpawnWindow(const char *lpWindowName, int nWidth, int nHeight)
 #else
 	SDL_DisplayMode mode;
 	SDL_GetDisplayMode(0, 0, &mode);
-	refreshDelay = 1000000 / mode.refresh_rate;
+	refreshDelay = 1000000 / (mode.refresh_rate ?: 60);
```
Comment (glebm) [inline SourceX/display.cpp]: What magic is this `?:`?

Comment (qndel) [reply SourceX/display.cpp]: indeed lol xD

Comment (qndel) [reply SourceX/display.cpp]: but it works as intended, seems to be shortened `(mode.refresh_rate ? mode.refresh_rate : 60)`
I'm impressed!

Comment (AJenbo) [reply SourceX/display.cpp]: It's common in the languages I'm used to, but looks like MSVC will have none of it :(
I might refactor this a bit to have things a bit more DRY.

**Pull request merged**